src/otx/algorithms/anomaly/adapters/anomalib/config/anomalib_config.py

```diff
@@ -41,6 +44,13 @@ def get_anomalib_config(task_name: str, otx_config: ConfigurableParameters) -> U
     """
     config_path = Path(anomalib.__file__).parent / "models" / task_name.lower() / "config.yaml"
     anomalib_config = get_configurable_parameters(model_name=task_name.lower(), config_path=config_path)
+
+    try:
+        anomalib_config.metrics.image.remove("AUROC")
+        anomalib_config.metrics.pixel.remove("AUROC")
+    except ValueError:
+        logger.info("Tried removing AUROC metric from anomalib config, but metric was not found.")
```

Comment:
Is the `AUROC`  the mandatory thing? 
If `torch.cumsum` included in the `AUROC` really matter, how about setting `deterministic=False` instead of `ValueError` if there is an `AUROC` metric  ?

Reply:
Is ok to remove "AUROC" metric? When I checked configuration.yaml in anomaly config directory, AUROC can be used as early stopping metric. Does early stopping work if user decides to use AUROC after this change is applied?